File: data/evaluation/geeks_for_geeks_successful_test_scripts/java/COMPUTE_AVERAGE_TWO_NUMBERS_WITHOUT_OVERFLOW.java
// Copyright (c) 2019-present, Facebook, Inc.
// All rights reserved.
//
// This source code is licensed under the license found in the
// LICENSE file in the root directory of this source tree.
//

import java.util. *;
import java.util.stream.*;
import java.lang.*;
import javafx.util.Pair;
public class COMPUTE_AVERAGE_TWO_NUMBERS_WITHOUT_OVERFLOW{
static int f_gold ( int a , int b ) {
  return ( a + b ) / 2 ;
}


//TOFILL

public static void main(String args[]) {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(1);
    param0.add(6);
    param0.add(75);
    param0.add(51);
    param0.add(19);
    param0.add(82);
    param0.add(72);
    param0.add(48);
    param0.add(12);
    param0.add(41);
    List<Integer> param1 = new ArrayList<>();
    param1.add(44);
    param1.add(61);
    param1.add(20);
    param1.add(17);
    param1.add(25);
    param1.add(98);
    param1.add(21);
    param1.add(41);
    param1.add(17);
    param1.add(80);
    for(int i = 0; i < param0.size(); ++i)
    {
        if(f_filled(param0.get(i),param1.get(i)) == f_gold(param0.get(i),param1.get(i)))
        {
            n_success+=1;
        }
    }
    System.out.println("#Results:" + n_success + ", " + param0.size());
}
}

File: data/evaluation/geeks_for_geeks_successful_test_scripts/python/COMPUTE_AVERAGE_TWO_NUMBERS_WITHOUT_OVERFLOW.py
# Copyright (c) 2019-present, Facebook, Inc.
# All rights reserved.
#
# This source code is licensed under the license found in the
# LICENSE file in the root directory of this source tree.
#
from math import floor

def f_gold ( a , b ) :
    return floor ( ( a + b ) / 2 )


#TOFILL

if __name__ == '__main__':
    param = [
    (1,44,),
    (6,61,),
    (75,20,),
    (51,17,),
    (19,25,),
    (82,98,),
    (72,21,),
    (48,41,),
    (12,17,),
    (41,80,)
        ]
    n_success = 0
    for i, parameters_set in enumerate(param):
        if f_filled(*parameters_set) == f_gold(*parameters_set):
            n_success+=1
    print("#Results: %i, %i" % (n_success, len(param)))